roles/ovb/bmc/tasks/os-net-config.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Register management network details
+  os_networks_facts:
+      cloud: "{{ provisioner.cloud | default(omit) }}" 
+      filters:
+          name: "{{ provisioner.prefix | default(omit) }}-management"
+  register: management_network_details
+- debug: var=management_network_details
+
+- name: Set fact for management network - default gateway
+  set_fact:
+      management_default_gateway: "{{ item.value[0].gateway_ip }}"
+  with_dict: "{{ management_network_details.ansible_facts }}"
+
+- name: Set fact for mengement subnet - CIDR prefix
+  set_fact:
+      management_cidr_prefix: "{{ item.split('/') | last }}"
```

Comment:
Can you use builtin [filters](http://docs.ansible.com/ansible/playbooks_filters_ipaddr.html) for this?